src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -180,14 +181,13 @@ def _upload_resources(clients: UploadClients, upload_state: UploadState) -> None
     namespaces = make_namespace_dict_from_onto_names(project_onto_dict)
 
     iri_lookup = IRILookups(
-        project_iri=project_iri,
+        project_iri=URIRef(project_iri),
         id_to_iri=upload_state.iri_resolver,
         jsonld_context=project_context,
```

Comment:
The json-ld context is no longer needed here, but it is for the stash. Therefore it remains for the time being.

Reply:
yes